text/0000-template.md

```diff
@@ -0,0 +1,159 @@
+# A Dynamic Configuration Service for the SDK
+
+This proposal is for an experiment to add a configuration service to dynamically configure metric  periods. Per-metric and tracing configuration is also intended to be added, with details left for a later iteration.
+
+It is related to [this pull request](https://github.com/open-telemetry/opentelemetry-proto/pull/155)
+
+## Motivation
+
+During normal use, users may wish to collect metrics every 10 minutes. Later, while investigating a production issue, the same user could easily increase information available for debugging by reconfiguring some of their processes to collect metrics every 30 seconds. Because this change is centralized and does not require redeploying with new configurations, there is lower friction and risk in updating the configurations.
+
+## Explanation
+
+This OTEP is a request for an experimental feature [open-telemetry/opentelemetry-specification#62](https://github.com/open-telemetry/opentelemetry-specification/pull/632). It is intended to seek approval to develop a proof of concept. This means no development will be done inside either the Open Telemetry SDK or the collector. Since this will be implemented mostly in the contrib repos, all of this functionality will be optional.
```

Comment:
will be optional. The implementation will be under http://go-contrib and http://.../collector-contrib.